layers/+lang/markdown/README.org

```diff
@@ -107,6 +108,8 @@ list of `("language" "mode")`:
 bunch of languages. Consider opening a PR on the Spacemacs repository to add
 new languages instead of overriding the variable in your dotfile.
 
+** Mixed pitched support
```

Comment:
Please also add a warning/note about the interaction with company.
See `Interaction with company` section on its repository.